packages/boa/tools/utils.js

```diff
@@ -13,7 +13,11 @@ if (BOA_TUNA && !BOA_CONDA_INDEX) {
   BOA_CONDA_INDEX = 'https://pypi.tuna.tsinghua.edu.cn/simple';
 }
 
-exports.run = (...args) => execSync.call(null, args.join(' '), { stdio: 'inherit' });
+exports.run = (...args) => {
+  const cmd = args.join(' ');
+  console.log(`sh "${cmd}"`);
```

Comment:
Is the log necessary?

Reply:
Yap, it's good for checking more information, see https://github.com/alibaba/pipcook/blob/726e1598fc74cf5f3fa3d65f78174307b70bd7a3/packages/boa/tools/utils.js#L25.

Reply:
Shall we only print it when DEBUG is set in env?

Reply:
This script is possible to run before the `install` command, thus `debug()` might be unavailable, and we could correct it in another PR.